src/search-field.tsx

```diff
@@ -15,6 +15,7 @@ import type {CommonFormFieldProps} from './text-field-base';
 export interface SearchFieldProps extends CommonFormFieldProps {
     onChangeValue?: (value: string, rawValue: string) => void;
     getSuggestions?: (value: string) => ReadonlyArray<string>;
+    inputMode?: string;
```

Comment:
this could be typed as:
`React.HTMLAttributes<HTMLInputElement>['inputMode']`

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L2958

Reply:
Yep, updated